docs/graphcache/gotchas.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Gotchas
+order: 5
+---
+
+# Gotchas
+
+When using the normalised cache there are a few out of the box features
+we need to be aware of, being aware of these features will make it easier
+to define best practices and to avoid pitfalls.
```

Comment:
```suggestion
Graphcache has a couple of hidden features behind the scenes, that handle how it deals with certain edge cases.
Being aware of these features and edge cases makes it easier to define best practices and to avoid pitfalls.
```